src/token/types/Token18.sol

```diff
@@ -62,7 +62,12 @@ library Token18Lib {
      * @param amount Amount of tokens to approve to spend
      */
     function approve(Token18 self, address grantee, UFixed18 amount) internal {
-        IERC20(Token18.unwrap(self)).safeApprove(grantee, UFixed18.unwrap(amount));
+        IERC20 token = IERC20(Token18.unwrap(self));
```

Comment:
Yeah, I agree with you that we shouldn't pull this safe logic in.

Reply:
removed in 3a84ed3a5e778619f8d68029a5257819368d2703